man/man8/execsnoop.8

```diff
@@ -56,6 +62,18 @@ Trace all exec() syscalls, and include timestamps:
 #
 .B execsnoop \-t
 .TP
+Display process UID:
+#
+.B execsnoop \-U
+.TP
+Trace only UID 1000:
+#
+.B execsnoop \-u 1000
+.TP
+Trace only processes launched by user and display UID column:
+#
+.B execsnoop \-Uu user
```

Comment:
Since this is an example, you need specific uid instead of `user`. The `execsnoop -Uu user` won't work.

Reply:
Well, as I wrote before, now this options also accepts user as argument too, in that case it will lookup UID in `/etc/passwd` (*or somewhere else, depends on configuration*) using `getpwnam`.